src/bounded_loop.rs

```diff
@@ -0,0 +1,151 @@
+use std::fmt;
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::collections::HashMap;
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::io::{BufReader, BufRead};
+use std::fs::File;
+
+lazy_static!{
+    static ref MUTAGEN_LOOP_COUNT: HashMap<LoopId, usize> = {
+        let mut h = HashMap::new();
+
+        File::open("target/mutagen/loops.txt").map(|file| {
+            let reader = BufReader::new(file);
+
+            for l in reader.lines() {
+                let line = l.unwrap_or(String::from(""));
+                let splits: Vec<&str> = line.split(",").collect();
+
+                if splits.len() != 2 {
+                    continue;
```

Comment:
Should we just throw away invalid input? Or perhaps at least warn the user?

Reply:
It appears we need yet another file for this. Ah well, let's do this in another PR.